IbisITK/itkVolumeReconstructionOpenCL/itkGPUVolumeReconstruction.hxx

```diff
@@ -160,7 +162,7 @@ GPUVolumeReconstruction< TImage >
 
   cl_program program;
   cl_kernel kernel = CreateKernelFromString(OriginalSourceString, cPreamble, kernelname, cOptions, &program);
-
+  delete[] OriginalSourceString;
```

Comment:
There should probably be an fclose() call somewhere around here. This code may not be used at the moment, but just to make sure.